### ISSUE TITLE
Ambient mode on DGPU

### ISSUE THREAD

**Issue by RebeenPJ** [seerge/g-helper#2011]
### Rules

- [X] I made myself familiar with the <a href='https://github.com/seerge/g-helper?tab=readme-ov-file'>Readme</a>, <a href='https://github.com/seerge/g-helper/wiki/FAQ'>FAQ</a> and <a href='https://github.com/seerge/g-helper/wiki/Troubleshooting'>Troubleshooting</a>.
- [X] I understand that, if insufficient information or no app logs will be provided, my issue will be closed without an answer.

### What's wrong?

The Ambient keyboard backlight mode doesn't seem to respond to changes on screen content when utilizing the DGPU on Standard mode

### How to reproduce the bug?

1. select Ambient mode for keyboard Backlight
2. Choose DGPU in Nvidia control panel (Advanced Optimus)

### Logs

[log.txt](https://github.com/seerge/g-helper/files/14122406/log.txt)

### Device and Model

ASUS TUF Gaming F15 FX507ZI4_FX507ZI

### Armoury Crate

Uninstalled

### Asus Services

None

### Version

0.148.0

### OS

Windows 11

**Comment by seerge** [comment]
@RebeenPJ hello, doesn't seem to change or doesn't change at all ? What it does is just makes a small screenshot of your display using windows API, and calculates average color of it. It doesn't depend anyhow on GPU mode

Does watching video like this one https://www.youtube.com/watch?v=QGVwbI3MUUU in full-screen reflect on keyboard ?

**Comment by RebeenPJ** [comment]
you're right sorry about that. i thought the issue was connected to gpu mode but after more testing that is not the case. maybe it is connected to fullscreen/borderless mode in games? or disconnecting charger that automatically pops up a window to close the gpu heavy programs that affects how the ambient mode works in the screenshot taking. any how im sorry for wasting your time you can disregard this thread.

**Comment by seerge** [comment]
@RebeenPJ 

It could be that some games or for example Netfix prevent screenshots from happening as a copyright measure or something :) It's normal.

> or disconnecting charger that automatically pops up a window to close the gpu heavy programs that affects how the ambient mode works in the screenshot taking.

This is absolutely not connected. You have ``Optimized`` GPU mode, and it's supposed to disable GPU when you unplug. But it checks for GPU usage before doing that (in order to prevent running game crashing for example). 

Closing as resolved then
